iron-lazy-pages.html

```diff
@@ -308,6 +317,7 @@
           this.timesFailed = 0;
         }
         var self = this;
+        var loadAsync = Polymer.dom(this).parentNode.loadAsync;
```

Comment:
Instead of resolving to the parentNode (which is potentially expensive) I would prefer to add a second argument to `_importHref` with the async option. This should be a new argument of `_loadAndShow` too which is called by the parent `iron-lazy-pages`.